tests/test_processdata_unittest.py

```diff
@@ -0,0 +1,100 @@
+import unittest
```

Comment:
Personal preference here, but would like to see these named "`tests/test_processdata.py`" rather than with the suffix `_unittest`.